cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   containers:
   - name: fluentd-elasticsearch
-    image: gcr.io/google_containers/fluentd-elasticsearch:1.17
+    image: aledbf/fluentd-elasticsearch:1.19
```

Comment:
Right. I'm just following the procedure defined in the [Makefile](https://github.com/aledbf/kubernetes/blob/a2e9b119f6e1e64d8c594437a7986b172d028641/cluster/addons/fluentd-gcp/fluentd-gcp-image/Makefile#L19)